src/inference/src/system_conf.cpp

```diff
@@ -225,6 +225,7 @@ std::vector<int> get_available_numa_nodes() {
 #    endif
 
 std::vector<std::vector<int>> get_proc_type_table() {
+    std::lock_guard<std::mutex> lock{cpu._proc_type_mutex};
```

Comment:
does `set_cpu_used` changes the global state?

Reply:
Yes, it fixed the reading data asynchronously issue.